File: Python/7kyu/ComplementaryDNA.py
import enum

dict = {
    "A": "T",
    "T": "A",
    "G": "C",
    "C": "G"
}


def DNA_strand(dna):
    return "".join([dict[d] for d in dna])


print(DNA_strand("ATTGC"))

File: JavaScript/7kyu/ComplementaryDNA.js
var translate = {
  A: "T",
  T: "A",
  G: "C",
  C: "G",
};

function DNAStrand(dna) {
  return dna
    .split("")
    .map((d) => translate[d])
    .join("");
}

function DNAStrand_2(dna) {
  return dna.replace(/./g, (w) => translate[w]);
}

console.log(DNAStrand_2("ATTGC"));
